### ISSUE TITLE
Activating Folding fangs does nothing.

### ISSUE THREAD

**Issue by Soyweiser** [CleverRaven/Cataclysm-DDA#22963]
Game version:
Tiles 7124

Operating system:
Win7

Tiles or curses:
Tiles

Mods active:
Base install.

### Expected behavior
Description of folding fangs says you can extend/retract them at will.

### Actual behavior
Can't be activated in the mutation menu. They apparently are always extended.

### Steps to reproduce the behavior
Give yourself the arachnid mutation (for the thresh), then folding fangs, open mutations menu and activate.

Thanks CantCatacl and DracoGriffin, for bringing it to my attention, and testing it resp. (also tested it myself).

**Comment by Rivet-the-Zombie** [comment]
Maybe we're assuming that they automatically pop out when needed?

**Comment by Soyweiser** [comment]
Yes could be, but then the json description is wrong. (And I assumed that it also had other errors, but seems it is just the description, and there is no 'activate=true' thing in the mutations).
